Update docs for download transformation (#112)
xataio/mdx-docs: 040-SDK/160-image-transformations.mdx

<TransformImageExample url="https://us-east-1.storage.xata.sh/4u1fh2o6p10blbutjnphcste94" options={{ contrast: 0.5 }} />

### download

> Option to force browser to download the image instead of displaying it. Value is the file name and extension is optional.

Reviewer: ```suggestion
### download

Depending upon the file type, browsers by default will handle URLs differently. Some will display files directly in the browser, while others will start an immediate download. Setting the `download` property will tell the browser to always download the file. The value allows you to assign a name, and extension to the file as it downloads. 
```

> Option to force browser to download the image instead of displaying it. Value is the file name and extension is optional.

Author: ```suggestion
Option to force browser to [download](https://us-east-1.storage.xata.sh/transform/download=xata-cat.jpg,format=jpeg/4u1fh2o6p10blbutjnphcste94) the image instead of displaying it. Value is the file name and extension is optional.
```

> Option to force browser to [download](https://us-east-1.storage.xata.sh/transform/download=xata-cat.jpg,format=jpeg/4u1fh2o6p10blbutjnphcste94) the image instead of displaying it. Value is the file name and extension is optional.

Author: ```suggestion
Depending on the file type, some browsers preview the file instead of downloading it. Setting the `download` property will make the browser to always download the file. The value allows you to assign a name, and optionally an extension to the file as it downloads.
```

Reviewer: Looks better :+1:

<TabbedCode tabs={['TypeScript', 'Python', 'URL']}>

```ts
const transformedImageUrl = transformImage(
  'https://us-east-1.storage.xata.sh/4u1fh2o6p10blbutjnphcste94',
  {
    download: "xata-cat.jpg",

>     format: "jpg"

Author: ```suggestion
    format: "jpeg"
```

  }
);
```

```python
img = xata.files().transform(
  "https://us-east-1.storage.xata.sh/4u1fh2o6p10blbutjnphcste94",
  {
    "download": "xata-cat.jpg",

>     "format": "jpg"

Author: ```suggestion
    "format": "jpeg"
```

  }
)
```

```txt

> https://us-east-1.storage.xata.sh/transform/download=xata-cat.jpg,format=jpg/4u1fh2o6p10blbutjnphcste94

Author: ```suggestion
https://us-east-1.storage.xata.sh/transform/download=xata-cat.jpg,format= jpeg/4u1fh2o6p10blbutjnphcste94
```

```

</TabbedCode>

> 

Author: ```suggestion

Example: Download [xata-cat.jpg](https://us-east-1.storage.xata.sh/transform/download=xata-cat.jpg,format=jpeg/4u1fh2o6p10blbutjnphcste94)

```

<TransformImageExample url="https://us-east-1.storage.xata.sh/4u1fh2o6p10blbutjnphcste94" options={{ contrast: 0.5 }} />

### download

> Depending on the file type, some browsers preview the file instead of downloading it. Setting the `download` property will make the browser to always download the file. The value allows you to assign a name, and optionally an extension to the file as it downloads.

Author: ```suggestion
Depending on the file type, some browsers preview the file instead of downloading it. Setting the `download` property will make the browser to always download the file. The value allows you to assign a name, and optionally an extension to the file that will be downloaded.
```
